docs/concepts/duplicate-legacy-metaos-add-ins.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Managing both a unified manifest and an XML manifest version of your Office Add-in
+description: Learn when and how to maintain versions of your add-in for each type of manifest.
+ms.topic: best-practice
+ms.date: 07/11/2023
+ms.localizationpriority: medium
+---
+
+# Managing both a unified manifest and an XML manifest version of your Office Add-in
+
+After you've created a version of your add-in that uses the [unified manifest for Microsoft 365](../develop/json-manifest-overview.md), you need to decide whether to maintain the existing add-in and, if so, what relationship it will have with the new version. In the long run, the best practice is to replace the existing add-in with the new version, but initially add-ins that support the unified manifest can only be installed on Microsoft 365 version 2309 ?????.????? and later. We're working hard to bring support to older versions of Microsoft 365. In the meantime, you need to maintain both versions.
```

Comment:
```suggestion
After you've created a version of your add-in that uses the [unified manifest for Microsoft 365](../develop/json-manifest-overview.md), you need to decide whether to maintain the existing add-in and, if so, what relationship it will have with the new version. In the long run, the best practice is to replace the existing add-in with the new version, but initially add-ins that support the unified manifest can only be installed on Microsoft 365 Version 2309 (Build ?????.?????) and later. We're working hard to bring support to older versions of Microsoft 365. In the meantime, you need to maintain both versions.
```